DataFormats/JetReco/src/classes_def_4.xml

```diff
@@ -137,4 +137,23 @@
    <class name="edm::Association<reco::FFTCaloJetCollection>" />
    <class name="edm::Wrapper<edm::Association<reco::FFTCaloJetCollection> >" />
 
+
+
```

Comment:
CATopJetProperties was moved to DataFormats/BTauReco
why is it coming up here? Need to remove